src/Connection.php

```diff
@@ -295,6 +295,10 @@ public function statement($query, $bindings = []): bool
         }
 
         // is DDL Query
+        // allow pretending for DDL queries
+        if($this->pretending())
+            return $this->affectingStatement($query, $bindings) !== null;
```

Comment:
I think runDdlBatch should be fixed to handle the pretending instead of forwarding it to affectingStatement.

Reply:
but isnt affectingStatement()s implementation of `pretend` is basically complete and simple and i would just end up duplicating affectingStatement() line for line, except for this range of lines

https://github.com/colopl/laravel-spanner/blob/master/src/Connection.php#L313-L323

Reply:
or is nothing in the affectingStatement method relevant?